docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   database:
-    image: "${DOCKER_REPO}genepi-devdb:latest"
+    image: "${DOCKER_REPO}genepi-devdb:czge-2022-04-04"
```

Comment:
This should go back to being `:latest`, right? We'll need to put in a new image for that tag that that has the data under the new PGDATA as part of this merge though.

Reply:
Yep!